doc/overview.rst

```diff
@@ -0,0 +1,12 @@
+=======================
+Overview of dbxincluder
+=======================
+
+dbxincluder is an implementation of the XInclude 1.1 specification (https://www.w3.org/TR/xinclude-11)
+with support for DocBook transclusion (http://docbook.org/docs/transclusion).
+
+
+Limitations
+===========
+
+It does not support the XInclude 1.1 "xpointer" attribute, as the XPointer functionality is currently not exposed by lxml.
```

Comment:
Well, the underlying reason why XPointer is not supported by dbxincluder **is** that it is not exposed by lxml.

The direct cause is rather "and reimplementing in Python instead of using libxml2 would be a waste of time".
Should I write that instead?